auth-api/src/auth_api/models/org.py

```diff
@@ -171,6 +171,20 @@ def search_org(cls, search: OrgSearch, environment: str):
             query = query.filter(Org.branch_name.ilike(f"%{search.branch_name}%"))
         if search.name:
             query = query.filter(Org.name.ilike(f"%{search.name}%"))
+        if search.member_search_text:
+            member_exists_subquery = text("""
+                EXISTS (
+                    SELECT 1
+                    FROM memberships
+                    JOIN users ON users.id = memberships.user_id
+                    WHERE memberships.org_id = orgs.id
+                    AND memberships.status = 1
+                    AND users.status = 1
+                    AND CONCAT(users.last_name, ' ', users.first_name, ' ', users.username) ILIKE :member_search_text
+                )
```

Comment:
Needed to use string subquery due to model cyclic dependencies. 
Membership model has an org import.

Reply:
Haha, this should be refactored to a service

Reply:
As long as there is a unit test should be ok, but I'll leave it up to you to decide

Reply:
I think we should make a ticket for the `sbc-connect` team to break alot of these dependencies as well?

Reply:
Yes, I started looking at doing a refactor, and it became a larger undertaking then I wanted for the scope of this ticket. Since it was just a subquery I kept it simple. 

I agree we should probably have some kind of tech debt ticket to clean this up.